From e490c780e77ca3834d47b05db2149af5339960e1 Mon Sep 17 00:00:00 2001
From: Alex Sarraf <asarraf@codeaurora.org>
Date: Wed, 21 Mar 2018 10:19:36 -0700
Subject: [PATCH 26/35] nile-common: Add vendor.hbtp service to init.qcom.rc

add vendor.hbtp service to init.qcom.rc and change the service name
in post_boot.sh from hbtp to vendor.hbtp

Change-Id: Ie072a9c385452e0ed8ab46bd760809ce8a6c8118
---
 rootdir/etc/init.qcom.post_boot.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/etc/init.qcom.post_boot.sh b/rootdir/etc/init.qcom.post_boot.sh
index ed7e03f..238097f 100755
--- a/rootdir/etc/init.qcom.post_boot.sh
+++ b/rootdir/etc/init.qcom.post_boot.sh
@@ -377,7 +377,7 @@ function start_hbtp()
         # Start the Host based Touch processing but not in the power off mode.
         bootmode=`getprop ro.bootmode`
         if [ "charger" != $bootmode ]; then
-                start hbtp
+                start vendor.hbtp
         fi
 }
 
-- 
2.17.1

